src/client.coffee

```diff
@@ -110,6 +110,14 @@ class Client
         cb(error, body)
     )
 
+  tasksStdout: (taskId, cb) ->
+    @api.tasksStdout(taskId, (error, body) ->
+      if not error?
+        cb(error, body)
+      else
+        cb(error, body)
```

Comment:
Maybe replace the "if - else" block with one line:
cb(error, body)

Reply:
Just following the style of the surrounding code.